pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>xmlpull-xpp3</groupId>
+  <artifactId>xmlpull-xpp3-parent</artifactId>
+  <!-- Use only 3 versions numbers (major, minor, hotfix) for OSGi compatibility -->
+  <version>1.1.4-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>XML Pull Parsing API + XPP3 parser parent</name>
+  <description>
+    General Pull parsing API + XPP3 parser implementation
+  </description>
+  <url>http://www.xmlpull.org/</url>
+  <inceptionYear>2002</inceptionYear>
+
+  <licenses>
+    <license>
+      <name>Public Domain</name>
+      <url>http://www.xmlpull.org/v1/download/unpacked/LICENSE.txt</url>
```

Comment:
I presume this is the same as what's now in Maven Central for earlier versions?

Reply:
Yes, see e.g. https://repo1.maven.org/maven2/xmlpull/xmlpull/1.1.3.4d_b4_min/xmlpull-1.1.3.4d_b4_min.pom

Reply:
Not sure about the xpp3 license at this point, we may want to remove this entry and just and specific entries in the sub modules.